tests/routes/test_sessions.py

```
@@ -748,6 +748,51 @@
+    assert response.status_code == 200
+    data = response.json()
+
+    # Validate response structure
+    assert "id" in data
+    assert data["id"] == session_id
+    assert "short_summary" in data
+    assert "long_summary" in data
+    # Summaries will be None since they're created asynchronously
+
+
+def test_get_session_summaries_nonexistent_session(
```

Ruff (0.12.2)

Missing return type annotation for public function `test_get_session_summaries_nonexistent_session`

Add return type annotation: `None`

(ANN201)